packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -145,10 +145,17 @@ module.exports = function ({ resources, options }) {
     https: protocol === "https",
     limit: limit,
     filter: excludePatterns && Array.isArray(excludePatterns) && function(req, res) {
-      return excludePatterns.some(glob => {
+      const pattern = !excludePatterns.some(glob => {
         isDebug && log.info(`Proxy request ${req.url} is matched to glob "${glob}": ${minimatch(req.url, glob)} ${providedBaseUri}`);
-        return !minimatch(req.url, glob);
+
+        return minimatch(req.url, glob);
       });
+      if (!pattern && req.url.includes("~") && (options.configuration && options.configuration.skipCache)) {
```

Comment:
why not just using the following command: 

`const newUrl = req.url.replaceAll(/\/~.*~\//g, "/");`